plugins/spu2-x/src/DecodeDPLII.cpp

```diff
@@ -1,269 +0,0 @@
-/* SPU2-X, A plugin for Emulating the Sound Processing Unit of the Playstation 2
- * Developed and maintained by the Pcsx2 Development Team.
- *
- * Original portions from SPU2ghz are (c) 2008 by David Quintana [gigaherz]
- *
- * This library is free software; you can redistribute it and/or modify it under
- * the terms of the GNU Lesser General Public License as published by the Free
- * Software Foundation; either version 2.1 of the License, or (at your
- * option) any later version.
- *
- * This library is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
- * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
- * for more details.
- *
- * You should have received a copy of the GNU Lesser General Public License along
- * with this library; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA
- */
-
-#include "Spu2.h"
-#include "DPLII.h"
```

Comment:
Not totally sure what DPLII is, but this is the only file that includes DPLII.h, so if it's going, the header probably should too.

Also, any files removed should be removed in CMakeLists.txt as well.

Reply:
Dolby Pro Logic II, anyway I only chose to remove this since it wasn't actually included neither in VS project files nor in cmake, didn't check the header but that can be removed as well since it's only called in this file.